clayui.com/content/docs/components/markup-menubar.md

```diff
@@ -9,19 +9,15 @@ order: 5
 <div class="nav-toc-absolute">
 <div class="nav-toc">
 
--   [Menubar Vertical Expand Md](#css-menubar-vertical-expand-md)
-    -   [Menubar Transparent](#css-menubar-vertical-expand-md-transparent)
-    -   [Menubar Decorated](#css-menubar-vertical-expand-md-decorated)
--   [Menubar Vertical Expand Lg](#css-menubar-vertical-expand-lg)
+-   [Menubar Vertical Expand Md(#css-menubar-vertical-expand-md)](#menubar-vertical-expand-mdcss-menubar-vertical-expand-md) - [Menubar Vertical Expand Md with Buttons](#menubar-vertical-expand-md-with-buttons)
+-   [Menubar Vertical Expand Lg(#css-menubar-vertical-expand-lg)](#menubar-vertical-expand-lgcss-menubar-vertical-expand-lg)
```

Comment:
Probably a format error 🙂

Reply:
Well... SF did this change... otherwise it was complaining. It also looks strange to me, but... 🤷‍♀️

Reply:
haha no problem, sometimes SF formats wrong when it has the wrong tab. I can send a commit correcting this without any problems because otherwise, the ToC will get a bug.

Reply:
Sure, go ahead!